stable/aws-for-fluent-bit/README.md

```diff
@@ -145,53 +148,63 @@ helm delete aws-for-fluent-bit --namespace kube-system
-|`opensearch.includeTagKey`| When enabled, it append the Tag name to the record. |`"Off"`|
-|`opensearch.tagKey`| When Include_Tag_Key is enabled, this property defines the key name for the tag. |`"_flb-key"`|
-|`opensearch.generateId`| When enabled, generate _id for outgoing records. This prevents duplicate records when retrying. |`"Off"`|
-|`opensearch.idKey`| If set, _id will be the value of the key from incoming record and Generate_ID option is ignored. | |
-|`opensearch.writeOperation`| Operation to use to write in bulk requests. |`"create"`|
-|`opensearch.replaceDots`| When enabled, replace field name dots with underscore. |`"Off"`|
-|`opensearch.traceOutput`| When enabled print the OpenSearch API calls to stdout (for diag only) |`"Off"`|
-|`opensearch.traceError`| When enabled print the OpenSearch API calls to stdout when OpenSearch returns an error (for diag only). |`"Off"`|
-|`opensearch.currentTimeIndex`| Use current time for index generation instead of message record |`"Off"`|
-|`opensearch.logstashPrefixKey`| When included: the value in the record that belongs to the key will be looked up and over-write the Logstash_Prefix for index generation. If the key/value is not found in the record then the Logstash_Prefix option will act as a fallback. Nested keys are not supported (if desired, you can use the nest filter plugin to remove nesting) | |
-|`opensearch.suppressTypeName`| When enabled, mapping types is removed and Type option is ignored. |`"Off"`|
-|`opensearch.extraOutputs`| Append extra outputs with value. This section helps you extend current chart implementation with ability to add extra parameters. For example, you can add [network](https://docs.fluentbit.io/manual/administration/networking) config like `opensearch.extraOutputs.net.dns.mode=TCP`. |`""`|
-|`additionalOutputs`| add outputs with value |`""`|
-|`priorityClassName`| Name of Priority Class to assign pods | |
-|`updateStrategy`| Optional update strategy |`type: RollingUpdate`|
-|`affinity`| Map of node/pod affinities |`{}`|
-|`env`| Optional List of pod environment variables for the pods |`[]`|
-|`tolerations`| Optional deployment tolerations |`[]`|
-|`nodeSelector`| Node labels for pod assignment |`{}`|
-|`annotations`| Optional pod annotations |`{}`|
-|`volumes`| Volumes for the pods, provide as a list of volume objects (see values.yaml) |  volumes for /var/log and /var/lib/docker/containers are present, along with a fluentbit config volume |
-|`volumeMounts`| Volume mounts for the pods, provided as a list of volumeMount objects (see values.yaml) | volumes for /var/log and /var/lib/docker/containers are mounted, along with a fluentbit config volume |
-|`dnsPolicy`| Optional dnsPolicy |`ClusterFirst`|
-|`hostNetwork`| If true, use hostNetwork |`false` |
+| `s3.externalId`| Specify an external ID for the STS API, can be used w ith the role_arn parameter if your role requires an external ID.
```

Comment:
I know you just edited this slightly here but there's a space between `w ith` which would be ideal to fix